exporter/templates/cat1/r3_1/_2.16.840.1.113883.10.20.24.3.2.xml

```diff
@@ -26,7 +26,7 @@
       </participantRole>
     </participant>
 
-    {{if hasReason .EntrySection.Entry}}
+    {{if .EntrySection.HasReason}}
```

Comment:
Should be `.EntrySection.Entry.HasReason` (This didn't break tests because we don't test R3.1)